frontend/src/utils/theme.tsx

```diff
@@ -1,20 +1,55 @@
-import { createTheme } from "@mui/material/styles";
+import { createTheme, PaletteColorOptions } from "@mui/material/styles";
 
 const colors = {
   blue: "#1472D3",
   orange: "#EF7C00",
+  white: "#FFFFFF",
+  grey: "#FAFAFA",
+  green: "#91CC75",
+  yellow: "#FAC858",
+  red: "#EE6666",
 };
 
+declare module "@mui/material/styles" {
+  interface Palette {
+    chart: PaletteColor;
+  }
+
+  interface PaletteOptions {
+    chart: PaletteColorOptions;
+  }
+
+  interface SimplePaletteColorOptions {
+    green?: string;
+    yellow?: string;
+    red?: string;
+  }
+
+  interface PaletteColor {
+    green?: string;
+    yellow?: string;
+    red?: string;
+  }
+}
+
 const theme = createTheme({
   typography: {
     fontSize: 16,
   },
   palette: {
```

Comment:
Let's do up a full colour palette with proper steps instead of the simplified colour palette. It would give us more flexibility that way.